src/__tests__/components/CreateTemplate/index.test.tsx

```diff
@@ -0,0 +1,74 @@
+import CreateTemplate from "@/components/CreateTemplate";
+import renderWithMockedProvider from "../../utils";
+import { fireEvent, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+describe("Create Template", () => {
+  it("Should render all page titles", () => {
+    renderWithMockedProvider(<CreateTemplate isOpen={true} onClose={() => void {}} />);
```

Comment:
This is the one definitely needs redux so renderWithMockedProvider is the best for this case.

Reply:
> This is the one definitely needs redux so renderWithMockedProvider is the best for this case.

fixed